common/primitives/src/parquet/column_compression_codec.rs

```diff
@@ -0,0 +1,32 @@
+use codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+use serde::{Deserialize, Serialize};
+use sp_std::prelude::*;
+
+/// Compression codecs: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md
+#[derive(
+	Clone, Encode, Decode, PartialEq, Debug, TypeInfo, Eq, MaxEncodedLen, Serialize, Deserialize,
+)]
+pub enum ColumnCompressionCodec {
```

Comment:
These enum is case sensitive. Add an attribute like: 
[#[serde(rename_all = "lowercase")]](https://serde.rs/container-attrs.html#rename_all) (or any other option)

Reply:
i think if anything we should either keep the casing we have or make everything uppercase.